src/Monolog/Handler/NewRelicHandler.php

```diff
@@ -104,8 +105,8 @@ protected function write(array $record)
             }
         }
 
-        if (isset($record['formatted']['extra']) && is_array($record['formatted']['extra'])) {
-            foreach ($record['formatted']['extra'] as $key => $parameter) {
+        if (isset($formattedRecord['extra']) && is_array($formattedRecord['extra'])) {
+            foreach ($formattedRecord['extra'] as $key => $parameter) {
```

Comment:
This won't actually work if you format to a string then formattedRecord['extra'] will not be normalized, it'll be the original data you passed to extra (or context above). That means if you pass in some data type that is not handled by the new relic code it'll error out. That's why the NormalizerFormatter is the default and the right choice IMO as it makes sure everything is normalized to common data-types.